package.json

```diff
@@ -65,15 +70,13 @@
     "spectron": "^11.1.0",
     "typescript": "~3.9.3",
     "vue-cli-plugin-electron-builder": "~2.0.0-rc.4",
+    "vue-cli-plugin-i18n": "^2.3.1",
     "vue-cli-plugin-vuetify": "~2.0.6",
     "vue-template-compiler": "^2.6.11",
     "vuetify-loader": "^1.3.0",
+    "webpack-dev-server": "^4.7.4",
```

Comment:
where is this used?

Reply:
this package is used for Cypress unit  testing configuration